### PR TITLE
consensus: consensus reactor handle message clean up

### PR DESCRIPTION
**Pull request by JayT106**
The voteSetBitsMessage doesn't need the ctx check because there is no peerMsg needs to be handled.

### CODE REVIEW

File: internal/consensus/reactor.go [hunk 1]
```diff
@@ -1400,10 +1399,6 @@ func (r *Reactor) processVoteSetBitsCh(ctx context.Context) {
 		envelope := iter.Envelope()
 
 		if err := r.handleMessage(ctx, r.voteSetBitsCh.ID, envelope); err != nil {
-			if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
```
Comment (tychoish) [inline internal/consensus/reactor.go:1403]: the other channels processed by handle message can return context errors, so we should leave htis.

**Review by tychoish — CHANGES_REQUESTED** [review]

**Comment by tychoish** [comment]
Modulo my comment this is fine, and I agree that it's not necessary to pass a context here, but I'm curious as what the inspiration of this change is, or if there's some underlying inspiration that I missed.

File: internal/consensus/reactor.go (hunk 1 above)
Comment (JayT106) [reply internal/consensus/reactor.go:1403]: If we check other channel msg handle logic like `processVoteCh`,  they are the same logic. Original I was thinking every handleMessage should check the context status immediately(the first commit in my PR), but actually the following `SendError` will check the context status, therefore I think this line can be removed.

**Comment by JayT106** [comment]
> Modulo my comment this is fine, and I agree that it's not necessary to pass a context here, but I'm curious as what the inspiration of this change is, or if there's some underlying inspiration that I missed.
The PR was trying the align the handleMessage logic within 4 channels, just think we don't have to pass context into `handleVoteSetBitsMessage` when checking how the reactor.handleMessage works.

File: internal/consensus/reactor.go (hunk 1 above)
Comment (JayT106) [reply internal/consensus/reactor.go:1403]: So the only difference just should we print the error log when the context error happens.

Comment (JayT106) [reply internal/consensus/reactor.go]: Ah, I am wrong. Roll back the changes.

**Pull request closed without merging**